src/components/appNavigation/stackNavigation.js

```diff
@@ -146,20 +200,75 @@ class AppView extends Component<{ descriptors: any, navigation: any, navigationC
     }
     const Component = getComponent(descriptor.getComponent(), { screenProps })
     const pageTitle = title || activeKey
+    const menu = <SideMenuPanel screenProps={screenProps} />
     return (
       <React.Fragment>
         <Helmet>
           <title>{`Good Dollar | ${pageTitle}`}</title>
         </Helmet>
         {!navigationBarHidden && <NavBar goBack={backButtonHidden ? undefined : this.pop} title={pageTitle} />}
-        <ScrollView contentContainerStyle={scrollableContainer}>
-          <SceneView navigation={descriptor.navigation} component={Component} screenProps={screenProps} />
-        </ScrollView>
+        <SideMenu menu={menu} menuPosition="right" isOpen={store.get('sidemenu').visible}>
+          <ScrollView contentContainerStyle={scrollableContainer}>
+            <SceneView navigation={descriptor.navigation} component={Component} screenProps={screenProps} />
+          </ScrollView>
+        </SideMenu>
       </React.Fragment>
     )
   }
 }
 
+const SideMenuPanel = () => {
+  const store = GDStore.useStore()
+  const API = useWrappedApi()
+  const [showDialogWithData] = useDialog()
+  return (
+    <View>
+      <View style={[styles.row, styles.closeIcon]} onClick={() => toggleSidemenu(store)}>
```

Comment:
dont use view with onclick its not react-native compatible
use touchableopacity with onPress